vault/audit.go

```diff
@@ -69,12 +72,15 @@ func (c *Core) enableAudit(entry *MountEntry) error {
 	}
 
 	// Generate a new UUID and view
-	entryUUID, err := uuid.GenerateUUID()
-	if err != nil {
-		return err
+	if entry.UUID == "" {
```

Comment:
Up to you, I'm not sure where this value would come from but `_, err := uuid.ParseUUID()`  could be used to ensure it's a correct uuid